Supporting accent characters in Search (#621)
Automattic/simplenote-ios: RELEASE-NOTES.txt

-   Search results now display matching tags!
-   We're now supporting the `tag:keyword` search operator. Simplenote will yield Notes whose tag names contain such *keyword*.
-   Notes List's Interface is now looking more beautiful than ever.

> -   Search is now diacritic insensitive: no need to type keywords with accents to get matches!

Reviewer: ```suggestion
-   Search is now diacritic insensitive: type search terms with or without accents to get matches!
```

@jleandroperez should we change this text to something like this ☝️ ?

Author: Thank you !!! Patching up as suggested 🔥
